devops/Makefile

```diff
@@ -19,6 +19,7 @@ export BACKUP_MONGODB_APP_NAME := $(or $(BACKUP_MONGODB_APP_NAME), mongodb-backu
 export BACKUP_CONFIGMAP_NAME := $(or $(BACKUP_CONFIGMAP_NAME), backup-conf)
 export BYPASS_CRA_INCOME_VERIFICATION := $(or $(BYPASS_CRA_INCOME_VERIFICATION), false)
 export BYPASS_APPLICATION_SUBMIT_VALIDATIONS := $(or $(BYPASS_APPLICATION_SUBMIT_VALIDATIONS), false)
+export SWAGGER_ENABLED := $(or ${SWAGGER_ENABLED}, false)
```

Comment:
Please do the same on the below file, but keep the default value as _true_.
sources\Makefile

Please notice that we have two make files. The change in _devops/Makefile_ is correct. The one that I am asking to be changed is _sources\Makefile_.